lib/compilers/dex2oat.ts

```diff
@@ -361,6 +390,197 @@ export class Dex2OatCompiler extends BaseCompiler {
         };
     }
 
+    // Gets a string of methods parameters in the form used in .smali and
+    // splits it.
+    // '[Ljava/lang/String;[IIZ' --> ['Ljava/lang/String;', '[I', 'I', 'Z']
+    splitMethodParameters(parameters: string): string[] {
```

Comment:
Not a must, but it would be nice to have some simple tests for this. I imagine we could also have use for this in the regular Java/Kotlin output

(e.g the constantpool references https://godbolt.org/z/5avxq578o)